container.css

```diff
@@ -0,0 +1,393 @@
+:root {
+    /* Color Definitions */
+    --bg-light: white; /* white for light theme navbar */
+    --bg-dark: #535455; /* dark theme navbar */
+    --text-color-light: #333; /* Dark text for light theme */

```

Comment:
--bg-dark: #141824;
--text-color-light: #6e7891;
 --text-color-dark: #666f87;
 These colors looks better, also i think you should make the text smaller in the top bar